pkg/trafficmngr/iptables/iptables_windows.go

```diff
@@ -29,14 +33,18 @@ type IPTables interface {
 	Exists(table string, chain string, rulespec ...string) (bool, error)
 }
 
-func (iptm IPTablesManager) SetupAndEnsureForwardRules(flannelIPv4Network ip.IP4Net, flannelIPv6Network ip.IP6Net, resyncPeriod int) {
+func (iptm IPTablesManager) Init(ctx context.Context, wg *sync.WaitGroup) error {
+	return trafficmngr.ErrUnimplemented
```

Comment:
same as in nftables, I am not 100% sure but I fear that if we return an error, flanneld.exe will stop immediatly